src/main/java/org/jooq/lambda/concurrent/Async.java

```diff
@@ -0,0 +1,20 @@
+package org.jooq.lambda.concurrent;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.Executor;
+import java.util.function.Supplier;
+
+public final class Async {
+    private Async() {
+
+    }
+
+    public static <U> CompletionStage<U> supplyAsync(Supplier<U> supplier, Executor executor) {
+        return SameExecutorCompletionStage.of(CompletableFuture.supplyAsync(supplier, executor), executor);
+    }
+
+    public static CompletionStage<Void> runAsync(Runnable runnable, Executor executor) {
+        return SameExecutorCompletionStage.of(CompletableFuture.runAsync(runnable, executor), executor);
+    }
```

Comment:
Ok sounds good.
Would you prefer this explicit style?
```
public static CompletionStage<Void> runAsync(Runnable runnable) {
    Executor executor = ForkJoinPool.commonPool();
    return SameExecutorCompletionStage.of(CompletableFuture.runAsync(runnable, executor), executor);
}
```

Or
```
public static CompletionStage<Void> runAsync(Runnable runnable) {
    return SameExecutorCompletionStage.of(CompletableFuture.runAsync(runnable), null);
}
```
This would make the decision of how to handle  null `Executor` a little more important.